databricks/sdk/dbutils.py

```diff
@@ -241,6 +243,75 @@ def __getattr__(self, util) -> '_ProxyUtil':
                           name=util)
 
 
+@dataclass
+class OverrideResult:
```

Comment:
Is this necessary? We just return `result.result` every time. Might as well return the result itself?

Reply:
This is to provide a type that wraps a None result from the override. Some of the functions can then safely return None, if the override is not found or is not able to run.